clusters/migrating-to-gcp.md

```diff
@@ -28,7 +28,8 @@ The documentation will be updated when Tokendings is publicly available.
 {% endhint %}
 
 ### Deploy
-Same mechanism as for on-premise clusters. See [GCP clusters][GCP].
+The same deployment mechanism is leveraged for both on-premise K8s clusters and the others non-on-premise.
+See [Deployment][deployment] section of the documentation for how to leverage the _NAIS deploy tool_.
```

Comment:
You can also just write `[deployment]`

Reply:
Thanks for the tip, learned something new! =)